### DISCUSSION TITLE
Inject Java Security Properties Bellsoft-Liberica

### DISCUSSION THREAD

**Discussion by tuckeremulls** [paketo-buildpacks/feedback#73] [Q&A]
I am attempting to set `networkaddress.cache.ttl` for a spring java application I am containerizing with bellsoft-liberica. I don't want to set the property in application code (and it can't be set with jvm runtime properties), but I can set it by adding the property to the `/conf/security/java.security` file. Is there any way I can point bellsoft-liberica to use this custom java security file where I am setting the property when it's building an image? Is there some sort of environment variable I can use or is there any way to inject it? Thanks!

**Comment by dmikusa** [comment]
Linking this to the discussion that was had over here -> https://github.com/paketo-buildpacks/libjvm/issues/259
